### PR TITLE
Adding new flags to CNI docs

### PR DESCRIPTION
**Pull request by stewartbutler**
Adding documentation about the new flags for the CNI repair code

### CODE REVIEW

**Comment by stewartbutler** [comment]
Depends on https://github.com/istio/cni/pull/255 and a PR that will be created after that merges to merge the changes into istio/istio

File: content/en/docs/setup/additional-setup/cni/index.md [hunk 1]
```diff
@@ -68,7 +68,14 @@ The following table shows all the options that the `istio-cni` configuration sup
 | `cniConfFileName` | | | Leave unset to auto-find the first file in the `cni-conf-dir` (as `kubelet` does).  Primarily used for testing `install-cni` plugin configuration.  If set, `install-cni` will inject the plugin configuration into this file in the `cni-conf-dir`. |
 | `psp_cluster_role` | | | This value refers to a `ClusterRole` and can be used to create a `RoleBinding` in the namespace of `istio-cni`. This is useful if you use [Pod Security Policies](https://kubernetes.io/docs/concepts/policy/pod-security-policy) and want to allow `istio-cni` to run as `priviliged` Pods. |
 | `podAnnotations` | | `{}` | Additional custom annotations to be set on pod level. |
-
+| `repair.enabled` | `boolean` | `true` | Enable or disable the [CNI Race Condition](https://github.com/istio/istio/issues/14327) detection and repair functionality. This adds `istio-validation` as an init container to all injected pods, which checks if a pod has had its CNI configuration correctly initialized by CNI. It also enables a new container in the CNI `DaemonSet` which monitors for pods and either labels or deletes them, per the values below.|
```
Comment (rlenglet) [inline content/en/docs/setup/additional-setup/cni/index.md]:
```suggestion

| `repair.enabled` | `boolean` | `true` | Enable or disable the [CNI Race Condition](https://github.com/istio/istio/issues/14327) detection and repair functionality. This injects an `istio-validation` init container into every injected pod, which checks if Istio CNI correctly initialized the pod's networking configuration. It also enables a new container in the CNI `DaemonSet` which monitors for pods and either labels or deletes them, per the values below.|
```

File: content/en/docs/setup/additional-setup/cni/index.md [hunk 2]
```diff
@@ -68,7 +68,14 @@ The following table shows all the options that the `istio-cni` configuration sup
...
+| `repair.hub` | | | The container registry to pull the `install-cni` image for the repair container. Defaults to the same as `hub`. |
+| `repair.tag` | | | The container tag to use to pull the `install-cni` image for the repair container. Defaults to the same as `tag`. |
+| `repair.initContainerName` | | `istio-validation` | An override for the init container name inspected by the repair controller, if you are using a non-standard pod injection configuration |
```
Comment (rlenglet) [inline content/en/docs/setup/additional-setup/cni/index.md]: End the description with a period.

File: content/en/docs/setup/additional-setup/cni/index.md [hunk 3]
```diff
@@ -68,7 +68,14 @@ The following table shows all the options that the `istio-cni` configuration sup
...
+| `repair.labelPods` | `boolean` | `true` | Instruct the repair controller to label pods it detects as uninitialized. Ignored if `deletePods` is true. |
```
Comment (rlenglet) [inline content/en/docs/setup/additional-setup/cni/index.md]: To be consistent: s/Instruct/Enable/
(same below)

**Pull request merged**